lib/grizzly/command_class/network_management_inclusion/node_add.ex

```diff
@@ -134,7 +138,7 @@ defmodule Grizzly.CommandClass.NetworkManagementInclusion.NodeAdd do
           }
         }
       ) do
-    {
+    _ = {
```

Comment:
Is this causing a Dialyzer error? This seems really strange to need a `_ = ` on a required return value.

Reply:
It does.

Reply:
What's the error?

Reply:
lib/grizzly/inclusion.ex:601:unmatched_return
Expression produces a value of type:

:ok | {:error, _}

but this value is unmatched.
________________________________________________________________________________
done (warnings were emitted)